examples/ssx-test-jwt-authentication/backend/index.ts

```diff
@@ -0,0 +1,48 @@
+import { SSXServer, SSXExpressMiddleware, SSXAuthenticationMethod } from '@spruceid/ssx-server'
+import express, { NextFunction, Request, Response } from 'express'
+import cors from 'cors'
+import { verify } from 'jsonwebtoken'
+
+/** Ideally this should live on .env file */
+const MY_SECRET_KEY = 'FAKESECRET'
```

Comment:
this does need to be in .env. you can create a .env.example with `MY_SECRET_KEY=` then add instructions in the README for the user to copy the file